app/assets/javascripts/models/story.js

```diff
@@ -62,55 +62,31 @@ var Story = module.exports = Backbone.Model.extend({
     model.setColumn();
   },
 
-  moveBetween: function(before, after) {
-    var beforeStory = this.collection.get(before);
-    var afterStory = this.collection.get(after);
-    var difference = (afterStory.position() - beforeStory.position()) / 2;
-    var newPosition = difference + beforeStory.position();
-    this.set({position: newPosition});
-    this.collection.sort();
+  move: function(previous_story_id, next_story_id) {
+    var newPosition = this.collection.calculateNewPosition(previous_story_id, next_story_id)
```

Comment:
What do you think about checking if it is in a collection to avoid breaking when it's not?

Maybe:

```js
if(this.collection) {
  const newPosition = this.collection.calculateNewPosition(previous_story_id, next_story_id);
  this.setPosition(newPosition);
}

return this;
```

Also, can we use camelCase for the parameters of the function?